Initial frontend config is misconfigured

**Describe the bug**
When setting the "frontend" environment (via adding the `-f docker-compose.frontend.yml` to `docker-compose` command) the frontend service exists with error:

```
npm ERR! path /var/www/public/package.json
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to project base directory
2. Run the docker-compose command:

```
docker-compose -f docker-compose.yml -f docker-compose.local.yml -f docker-compose.ssl.yml -f docker-compose.frontend.yml up -d
```
3. Check `frontend` service logs with `docker-compose -f docker-compose.yml -f docker-compose.local.yml -f docker-compose.ssl.yml -f docker-compose.frontend.yml logs -f frontend`

4. Note the error stack trace:

```
frontend_1  | npm ERR! path /var/www/public/package.json
frontend_1  | npm ERR! code ENOENT
frontend_1  | npm ERR! errno -2
frontend_1  | npm ERR! syscall open
frontend_1  | npm ERR! enoent ENOENT: no such file or directory, open '/var/www/public/package.json'
frontend_1  | npm ERR! enoent This is related to npm not being able to find a file.
frontend_1  | npm ERR! enoent
frontend_1  |
frontend_1  | npm ERR! A complete log of this run can be found in:
frontend_1  | npm ERR!     /root/.npm/_logs/2019-03-11T10_46_46_105Z-debug.log
frontend_1  | npm ERR! path /var/www/public/package.json
frontend_1  | npm ERR! code ENOENT
frontend_1  | npm ERR! errno -2
frontend_1  | npm ERR! syscall open
frontend_1  | npm ERR! enoent ENOENT: no such file or directory, open '/var/www/public/package.json'
frontend_1  | npm ERR! enoent This is related to npm not being able to find a file.
frontend_1  | npm ERR! enoent
frontend_1  |
frontend_1  | npm ERR! A complete log of this run can be found in:
frontend_1  | npm ERR!     /root/.npm/_logs/2019-03-11T10_46_46_337Z-debug.log
```

**Expected behavior**
Fronted service compiles the application successfully.

**Potential solution**
After investigating the `docker-compose.frontend.yml` turned out, the incorrect folder has been mapped to the container. 

The change of the mapped path to proposed in [docs](https://github.com/scandipwa/scandipwa-base/blob/master/docs/F-Frontend-container.md) does not resolve the issue completely. The new issue related to files appears:

```
frontend_1  | ERROR in ./src/app/style/main.scss
...
frontend_1  | ModuleBuildError: Module build failed (from ./node_modules/sass-resources-loader/lib/loader.js):
frontend_1  | Error:
frontend_1  |       Something wrong with provided resources.
frontend_1  |       Make sure 'options.resources' is String or Array of Strings.
frontend_1  |
...
frontend_1  | ERROR in ./src/app/route/index.js
frontend_1  | Module not found: Error: Can't resolve '../component/Breadcrumbs' in '/var/www/public/src/app/route'
...
```

--- a/docker-compose.frontend.yml
+++ b/docker-compose.frontend.yml
@@ -13,8 +13,9 @@ services:
         - BASEPATH=${BASEPATH}
         - NODEJS_VERSION=${NODEJS_VERSION}
     volumes:
-      - ./src/localmodules/scandipwa-base-theme:/var/www/public
-    working_dir: "/var/www/public"
+      - ./src/app/design/frontend/Scandiweb/pwa:/var/www/public/app/design/frontend/Scandiweb/pwa
+      - ./src/vendor/scandipwa/source:/var/www/public/vendor/scandipwa/source
+    working_dir: "/var/www/public/app/design/frontend/Scandiweb/pwa/"
     env_file: .env
     expose:
       - 3003

